manifests/server/instance/systemd.pp

```diff
@@ -0,0 +1,26 @@
+# @summary This define handles systemd drop-in files for the postgres main instance (default) or additional instances
+# @param service_name Overrides the default PostgreSQL service name.
+# @param drop_in_ensure sets the Systemd drop-in file to present or absent
+# @api private
+define postgresql::server::instance::systemd (
+  Variant[String[1], Integer] $port,
+  Stdlib::Absolutepath $datadir,
+  String[1] $extra_systemd_config,
```

Comment:
Isn't this optional? Or at least should be:
```suggestion
  Optional[String[1]] $extra_systemd_config = undef,
```

Reply:
committed. I will add this to my review in #1397